variableForUserParams.proto

```diff
@@ -0,0 +1,73 @@
+syntax = "proto3";
```

Comment:
When we figure out how to automatically pull in the WASM updates into these SDKs, we should make sure this gets automatically updated as well.

Reply:
Yeah, i pushed that out to a separate task (DVC-7414). I fought with Gradle for a couple hours yesterday with no success so I kicked the issue forward.